crates/dbs-boot/src/x86_64/mod.rs

```diff
@@ -118,6 +132,63 @@ pub fn initrd_load_addr<M: GuestMemory>(guest_mem: &M, initrd_size: u64) -> Resu
     Ok(align_to_pagesize(lowmem_size - initrd_size))
 }
 
+/// Helps to initialize page table information for vcpu.
+pub fn setup_page_tables<M: GuestMemory>(
+    mem: &M,
+    sregs: &mut kvm_sregs,
+    existing_pgtable: Option<GuestAddress>,
+) -> Result<Option<GuestAddress>, Error> {
```

Comment:
We could return `GuestAddres` instead of `Option<GuestAddress>`

Reply:
Changed. Thanks:)